backend/src/chats/message/entities/message.entity.ts

```diff
@@ -25,13 +26,19 @@ export class Message {
 	@UpdateDateColumn()
 	updated_at: Date;
 
-	@ManyToOne((type) => User, (user) => user.id)
+	@ManyToOne(() => User, (user) => user.id, { nullable: false })
 	@JoinColumn({ name: 'sender_id' })
 	sender_id: User;
 
-	@ManyToOne((type) => Chat, (chat) => chat.id)
-	@JoinColumn({ name: 'chat_id' })
-	chat_id: Chat;
+	@RelationId((message: Message) => message.sender_id)
+	user_id: number;
+
+	@ManyToOne(() => Chat, (chat) => chat.id, { nullable: false })
+	@JoinColumn({ name: 'chat' })
+	chat: Chat;
```

Comment:
is this a chat_id? probably nottt?

Reply:
is what a chat_id? this column name? I renamed it to 'chat', since it's a full relation, and when loaded, resolves to a full chat object. And added a chat_id property (just below this one) with the @RelationId decorator to save the ID